### PR TITLE
Feature/uar 207 oe_number / entity_number

### PR DESCRIPTION
**Pull request by AndreKramer7654**
https://companieshouse.atlassian.net/browse/UAR-207 Request on Costs PR to use entity_number instead of oe_number as this aligns with entity_name.

### CODE REVIEW

**Review by olaych — APPROVED** [review]
Looks good to me for oe_number changed to entity_number

**Review by bantao-ch — COMMENTED** [review]
Would we now need to update the "What is an OE Number" page to reflect these changes to "What is the Entity Number" ? -- in the page html ?

**Comment by AndreKramer7654** [comment]
> Would we now need to update the "What is an OE Number" page to reflect these changes to "What is the Entity Number" ? -- in the page html ?

No need to change user facing

**Comment by AndreKramer7654** [comment]
> Would we now need to update the "What is an OE Number" page to reflect these changes to "What is the Entity Number" ? -- in the page html ?

No need to change user facing

**Review by bantao-ch — APPROVED** [review]

File: src/validation/overseas.entity.query.validation.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ErrorMessages } from "./error.messages";
 import { VALID_OE_NUMBER_FORMAT } from "./regex/regex.validation";
 
 export const overseasEntityQuery = [
-  body("oe_number")
+  body("entity_number")
     .not().isEmpty({ ignore_whitespace: true })
     .withMessage(ErrorMessages.OE_QUERY_NUMBER)
     .matches(VALID_OE_NUMBER_FORMAT)
```
Comment (corybudd-ch) [inline src/validation/overseas.entity.query.validation.ts:10]: Should we change the naming of OE_QUERY_NUMBER and VALID_OE_NUMBER_FORMAT as well?

**Comment by corybudd-ch** [comment]
Also whenever merging this definitely use Github's squash and merge as there are 52 commits

File: src/validation/overseas.entity.query.validation.ts (hunk 1 above)
Comment (AndreKramer7654) [reply src/validation/overseas.entity.query.validation.ts:10]: No. There will be design input coming. This is just to allow cost to flow through to api.

**Review by corybudd-ch — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Pull request merged**